src/v2i-hub/ODEForwardPlugin/src/ODEForwardPlugin.cpp

```diff
@@ -218,9 +218,9 @@ void ODEForwardPlugin::sendBsmKafkaMessage(BsmMessage &msg, routeable_message &r
 }
 
 void ODEForwardPlugin::sendUDPMessage(routeable_message &routeableMsg, UDPMessageType udpMessageType) const{
-	std::string message = routeableMsg.get_payload_str().c_str();
-	PLOG(logDEBUG) << "Sending UDP Message: " << message;
-	_udpMessageForwarder->sendMessage(udpMessageType, message);
+	std::string hex = routeableMsg.get_payload_str().c_str();
+	_udpMessageForwarder->sendMessage(udpMessageType, hex);
+	PLOG(logDEBUG) << "Sent UDP " + _udpMessageForwarder->toString(udpMessageType)+ " Hex: " << hex;
```

Comment:
I think plugin client will log outgoing messages at DEBUG1 or DEBUG2 so I think this is probably a redundant log message.

Reply:
This log also print which type of message the  UDP hex is about. The plugin client might not be able to provide this info.

Reply:
Sounds good to me